code/controllers/subsystem/polling.dm

```diff
@@ -85,7 +85,7 @@ SUBSYSTEM_DEF(polling)
 		// Opt-out for admins whom are currently adminned.
 		if(role && (!candidate_mob.client.prefs.read_preference(/datum/preference/toggle/ghost_roles_as_admin)) && candidate_mob.client.holder)
 			continue
-		if(role && !is_eligible(candidate_mob, role, check_jobban, ignore_category))
+		if(!is_eligible(candidate_mob, role, check_jobban, ignore_category))
```

Comment:
honestly should we remove `role` from all of these? the whole point of the `ghost_roles_as_admin` framework is so you wouldn't be inundated with stuff you didn't want, and the bypass for stuff like monkey helmets just seems to completely defeat the purpose since it will still steal focus away from what the admin is doing (i think it's changed but still, feels silly)

Reply:
done